sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/aio/_text_analytics_client_async.py

```diff
@@ -643,6 +643,9 @@ async def begin_analyze_healthcare_entities(  # type: ignore
         :keyword str model_version: This value indicates which model will
             be used for scoring, e.g. "latest", "2019-10-01". If a model-version
             is not specified, the API will default to the latest, non-preview version.
+            Currently not working on the service side at time of release, as service will
+            only use the latest model. Service is aware, and once it's been fixed on the service
+            side, the SDK should work automatically.
```

Comment:
I am not sure if this should be in the docstring itself. @maririos what do you think?

Reply:
I think it should be since most users will look at docstrings instead of changelog for errors. I also believe @maririos has added known issues to her docstrings in recent PRs

Reply:
Yes but once the service fixes the issue, users could be confused, especially first timers. I think it is fair to expect our users to read the readme file. Which made me realize you did not add the known issue section there but to changelog. I think you should at least add it to the readme file.